src/Frontend/Page/Idea.hs

```diff
@@ -68,7 +68,11 @@ instance ToHtml ViewIdea where
     -- The result might be that too much information is displayed.
     toHtml p@(ViewIdea idea phase) = semanticDiv p $ do
         header_ [class_ "detail-header"] $ do
-            a_ [class_ "btn m-back detail-header-back", href_ U.Broken] "Zum Thema"
+            a_ [ class_ "btn m-back detail-header-back"
```

Comment:
> Unrelated to this change, but why is it 'btn', but `a_` instead of not  `button_`?

My guess is that the css class 'btn' makes it *look* like a button, whereas the anchor element `a_` is better because it doesn't have certain undesired effects.  For instance, there can only be one `button_` per form, and its mechanics are very specific.